src/pieces/gui.py

```diff
@@ -78,10 +78,10 @@ def print_help():
     print("Available Commands:")
     print("  run             - Starts a looped version of the CLI that only requires you to type the flag")
     print()
-    print("  list            - List all assets")
-    print("  list x          - Lists maximum of x number of assets")
-    print("  list apps       - List all registered applications")
-    print("  list models     - List all registered AI models")
+    print("  list            - List all assets defaults to 10 assets")
+    print("  list_assets x   - Lists maximum of x number of assets")
+    print("  list_apps       - List all registered applications")
+    print("  list_models     - List all registered AI models")
```

Comment:
I would still like the commands to be "list assets" and "list apps" and "list models",

so lets move those back to how those were before but good job splitting up the code into smaller more digestible chunks.

I would also like to see the tests get updated/created for these commands to reflect the changes that you made in here.(this should also be a trend w/ each pr moving forward). IE will want a test with each feature/major adjustment to the code if there is not already one.